### PR TITLE
Google oauth enhancements

### PR DESCRIPTION
**Pull request by andkamau**

### CODE REVIEW

File: aleph/templates/layout.html [hunk 1]
```diff
@@ -4,6 +4,11 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="google" content="notranslate" />
+    
+    <meta name="google-signin-scope" content="profile email">
+    <meta name="google-signin-client_id" content="914424606018-68qjso1v76sath3t3pe639oo42qopddg.apps.googleusercontent.com">
```
Comment (DavidLemayian) [inline aleph/templates/layout.html]: This should probably be `env` set.

File: aleph/default_settings.py [hunk 2]
```diff
@@ -56,3 +56,6 @@
     'access_token_url': 'https://accounts.google.com/o/oauth2/token',
     'authorize_url': 'https://accounts.google.com/o/oauth2/auth',
 }
+
+PROVIDER_CLIENT_ID = '914424606018-68qjso1v76sath3t3pe639oo42qopddg.apps.googleusercontent.com'
```
Comment (DavidLemayian) [inline aleph/default_settings.py]: This should probably be `env` set.

**Comment by pudo** [comment]
Just for me to clarify: what is the advantage of using a specific library for google support, rather than just generic OAuth support?

**Comment by pudo** [comment]
@andkamau - first off: thanks for the pull request! :) I've just pushed a few basic changes to the config which should enable google login, but it's just using the ``oauthlib`` library. Would be cool to see if there's a broader rationale to adopt the more specific packages you propose. See: #e43643b872139f07306348712a6c8e88fcc0684c

**Comment by pudo** [comment]
Check out http://aleph.pudo.org/#/ - seems like OAuth via Google is working. This is based on https://github.com/pudo/aleph with just changes in the ``aleph.env`` configuration.

**Comment by andkamau** [comment]
hi @pudo . thanks for that!
i was tasked with implementing google login. other than that, no inherent reason to go with flask-googlelogin over generic oauth.
cc. @DavidLemayian

**Comment by pudo** [comment]
Hey @andkamau -- good to know. I'm really keen to make aleph more re-usable, need to write a ton of documentation and build in theming etc. But I think you did really cool work on enabling more types of logins :)

**Pull request closed without merging**